charts/workflows/charts/move2kube/templates/01-configmap_01-m2k-resources.yaml

```diff
@@ -0,0 +1,2070 @@
+              in: query
+              description: retrieve either logged-in user messages, system messages or both
+              required: false
+              schema:
+                type: string
+                enum:
+                  - all
+                  - user
+                  - system
+            - name: read
+              in: query
+              description: Notifications read or not
+              required: false
+              schema:
+                type: boolean
+          responses:
+            '200':
+              description: Successful operation
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      count:
+                        type: number
+                    required: ['count']
+      /notifications/read:
+        put:
+          tags:
+            - notifications
+          summary: Set notification as read/unread
+          description: Set notification as read/unread
+          operationId: setRead
+          parameters:
+            - name: messageId
+              in: query
+              description: The message ID
+              required: true
+              schema:
+                type: string
+            - name: read
+              in: query
+              description: read/unread
+              required: true
+              schema:
+                type: boolean
+          responses:
+            '200':
+              description: Successful operation
+    components:
+      schemas:
+        Notifications:
+          type: array
+          items:
+            $ref: '#/components/schemas/Notification'
+        Notification:
+          properties:
+            id:
+              type: string
+            created:
+              type: string
+              format: date-time
+            readByUser:
+              type: boolean
+            isSystem:
+              type: boolean
+            origin:
+              type: string
+            title:
+              type: string
+            message:
+              type: string
+            topic:
+              type: string
+            actions:
+              type: array
+              items:
+                $ref: '#/components/schemas/Action'
+          required: [id, created, readByUser, isSystem, origin, title, actions]
+        Action:
+          properties:
+            id:
+              type: string
+            title:
+              type: string
+            url:
+              type: string
+          required: [id, title, url]
+        CreateBody:
+          properties:
+            origin:
+              type: string
+            title:
+              type: string
+            message:
+              type: string
+            topic:
+              type: string
+          required: [origin, title]
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: 01-m2k-resources
+  namespace: default
```

Comment:
we should get read of this namespace, or at least modify them all to sonataflow-infra to make them work nicely with the existing deployment

Reply:
btw, this should have been supported by the kn-workflow gen-manifests